File: services/mono/Bacera.Gateway.Core/Types/AccountTagTypes.cs
namespace Bacera.Gateway;

public class AccountTagTypes
{
    public const string DailyConfirmEmail = "DailyConfirmEmail";
    public const string DefaultAgentAccount = "DefaultAgentAccount";
    public const string DefaultSalesAccount = "DefaultSalesAccount";
    public const string NewRebateTest = "NewRebateTest";
    public const string Test = "Test";
    public const string SwapFree = "SwapFree";
    public const string AddPips = "AddPips";
    public const string AddCommission = "AddCommission";
    public const string AutoCreate = "AutoCreate";
    public const string AutoCreateConfirmed = "AutoCreateConfirmed";


    public static readonly string[] All =
    {
        DailyConfirmEmail,
        NewRebateTest,
        Test,
        SwapFree,
        AddPips,
        AddCommission,
        DefaultAgentAccount,
        DefaultSalesAccount,
        AutoCreate,
        AutoCreateConfirmed
    };
}

File: app/web/vue/src/core/types/AccountTagTypes.ts
export const AccountTagTypes = {
  // Unknown: "Unknown",
  DailyConfirmEmail: "DailyConfirmEmail",
  Test: "Test",
  AddPips: "AddPips",
  SwapFree: "SwapFree",
  AddCommission: "AddCommission",
  PauseReleaseRebate: "PauseReleaseRebate",
};

export const getAllAccountTagTypes = (keys: string[]) =>
  Object.keys(keys ?? AccountTagTypes).map((key) => ({
    label: key,
    value: AccountTagTypes[key],
  }));
